dtos/systemevent.go

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+ * Copyright 2022 Intel Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package dtos
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos/common"
+)
+
+// SystemEvent defines the data for a system event
+type SystemEvent struct {
+	common.Versionable `json:",inline"`
+	Type               string
+	Action             string
+	Source             string
+	Owner              string
+	Tags               map[string]string
+	Details            any
+	Timestamp          int64
```

Comment:
YES, forgot the json attribute tags! THX!

Reply:
Do we need validation. IMO, no since only our code is generating this DTO. Thoughts?

Reply:
it's fine to go without the validation, because, not like REST, there is no place to send the error response and the validation rule would various from different system event.